src/components/shared/currency/ETHAmount.tsx

```diff
@@ -24,18 +20,6 @@ export default function ETHAmount({
   precision?: number
   padEnd?: boolean
 }) {
-  const isMaxUint =
-    BigNumber.isBigNumber(amount) &&
-    (amount.eq(constants.MaxUint256) || amount.eq(MAX_DISTRIBUTION_LIMIT))
-
-  if (isMaxUint) {
-    return (
-      <Tooltip title={t`Distribution limit infinite`}>
-        <Trans>No limit</Trans>
```

Comment:
"No limit" doesn't apply to all uses of `ETHAmount`. Moved the check outside the component and it seemed to fix the issue reported in Discord.

Reply:
Ok sweet